mlflow/environment_variables.py

```diff
@@ -566,3 +566,13 @@ def get(self):
 MLFLOW_UNITY_CATALOG_PRESIGNED_URLS_ENABLED = _BooleanEnvironmentVariable(
     "MLFLOW_UNITY_CATALOG_PRESIGNED_URLS_ENABLED", False
 )
+
+#: Private configuration option.
+#: Enables the ability to catch exceptions within MLflow evaluate for classification models
+#: where a class imbalance due to a missing target class would raise an error in the
+#: underlying metrology modules (scikit-learn). If set to True, specific exceptions will be
+#: caught, alerted via the warnings module, and evaluation will resume.
+#: (default: ``False``)
+_MLFLOW_EVALUATE_CLASSIFICATION_ERRORS_WARN_ONLY = _BooleanEnvironmentVariable(
```

Comment:
nit: Related to the comment below, if we plan to use `log_warning` param in the future, can we rename this to `_MLFLOW_EVALUATE_SUPPRESS_CLASSIFICATION_ERRORS` as we don't always emit warning?

Reply:
Good call :) updating!